src/test/java/uk/gov/ons/ssdc/caseprocessor/messaging/SurveyLaunchedReceiverIT.java

```diff
@@ -36,7 +36,9 @@
   private static final UUID TEST_CASE_ID = UUID.randomUUID();
   private static final String TEST_QID = "1234334";
   private static final String TEST_UAC = "9434343";
-  private static final String RH_CASE_QUEUE = "case.rh.case";
+
+  @Value("${queueconfig.outbound-case-queue}")
+  private String RH_CASE_QUEUE;
```

Comment:
all-caps is for constants... which I think this should be... it's not prod config